### ISSUE TITLE
Attention mask and  pad token id warning in videochat2_mistral

### ISSUE THREAD

**Issue by YajieW99** [OpenGVLab/Ask-Anything#183]
Some warning appears when the model outputs the answer. How can I deal with it?
`The attention mask and the pad token id were not set yet. As a consequence, you may observe unexpected behavior. Please pass your input's 'attention_mask' to obtain reliable results. Setting 'pad_token_id' to 'eos_token_id':2 for open-end generation.`

**Comment by Andy1621** [comment]
You can just ignore the warnning.

**Comment by yinanhe** [comment]
Hi, we will close this issue.

Feel free to contact us if you have other questions.
